### DISCUSSION TITLE
Support for multiple JACK outputs and JACK MIDI inputs

### DISCUSSION THREAD

**Discussion by grammoboy2** [sfztools/sfizz#1213] [Requests]
The nice thing about Linuxsampler, is that it allows for advanced routing, as is described in this document for instance. You can add multiple JACK audio outputs and JACK MIDI inputs. Would be nice for Sfizz as well.

https://github.com/ccherrett/oom/wiki/Linuxsampler

**Comment by paulfd** [comment]
We have some thoughts on this in #9, related to multiple outputs as specified in sfz instruments. As for multiple midi inputs it feels out of scope---I think we could argue that the VST/LV2 plugins pull in too many dependencies and could live in their own repository.

**Comment by grammoboy2** [comment]
From that openoctave project, there was a professional orchestral composer, who had everything set up with a 1:1 routing I think. One instrument, one midi input, one JACK (stereo?) output. Probably to get the best result without data interrupting each other. But I don't know the technical side of it exactly. 

I think I tried to more or less replicate such a setup years ago:
https://github.com/grammoboy2/template_oom/blob/master/LS_Studio_Final_oom2_test3_1.lscp
